jobs/build/ocp4/Jenkinsfile

```diff
@@ -152,6 +152,8 @@ node {
         currentBuild.description = ""
     }
 
+    def job_id = "ocp4-${currentBuild.number}"
```

Comment:
Can we get a full link? 
I'd like to be able to click on the reference, also when the lock is claimed from, say, a hack space.
Or maybe `$BUILD_URL` should be taken.